packages/core/lib/Client.ts

```diff
@@ -87,49 +82,44 @@ export class Client {
 
   // -----------------------
 
+  public async setRelease(release: string) {
+    const adapter = await this.awaitAdapter();
+    await adapter.setRelease(release);
+    return this;
+  }
+
   // ---------------- CONTEXT
 
-  public setUserContext(user?: IUser) {
+  public getContext() {
+    // TODO: check for cyclic objects
```

Comment:
https://github.com/getsentry/raven-js/blob/master/vendor/json-stringify-safe/stringify.js

Reply:
Thanks for the review, so about this function. I wasn't able to import the existing implementation and I want to have a fancy typescript and also thinking about if it would be possible to prevent circular refs upon setting it instead of reading.